data/deep missions.txt

```diff
@@ -744,39 +745,42 @@ mission "Deep: Singularity 0A"
 			`	"Because we want to know what you were doing with Albert Foster."`
 			choice
 				`	(Follow their orders.)`
-				`	(Close the hatch and flee)`
+				`	(Close the hatch and flee.)`
 					goto flee
 			
 			label orders
 			`	You follow the officers to a building located on the eastern edge of the spaceport. You are led to a small, dimly lit room on the second floor that contains nothing but two chairs and a table. You are told to sit in one of the chairs and wait.`
 				goto questioning
 			
 			label flee
-			`	You attempt to close the hatch, but suddenly begin cunvolsing and drop to the floor. Before you black out, you spot one of the officers holding a stun gun.`
+			`	You attempt to close the hatch, but a strong tingling sensation radiates from your abdomen. As you drop to the floor, convulsing, you hold your eyes open long enough to glimpse a stun gun in one officer's hand.`
 			``
-			`	You wake up sitting in a chair in a dimly lit room. Across the table in front of you is another chair, but it is empty.`
+			`	You wake up, slumped in a chair, in a dimly lit room. Across the table is another chair, empty. As you twist about, examining the rest of the room, the lingering soreness in your abdomen reminds you that even though you are alone for the moment, attempting to leave is likely unwise.`
 			
 			label questioning
-			`	Ten minutes pass in silence until a figure enters the room. The figure passes into the light to reveal a tall dark woman in what looks like a Deep Security uniform, but with a number of distinct differences to it.`
+			choice
+				`	(Wait.)`
```

Comment:
I don't see the need for this empty choice; it's not as if there was a large amount of dialog around this choice so something needed to break it up.

Reply:
More food-for-thought / future options than anything - the player has been instantly accosted upon returning to Deep space after being chased out of it, so perhaps there is a desire for a branch where the player attempts to escape instead of waiting.

Reply:
*"You try to escape, but you're chained to the chair, which is bolted to the ground."*